docs/_docs/permalinks.md

```diff
@@ -280,38 +294,76 @@ Given a post named: `/2009-04-29-slap-chop.md`
+
+Collections have their own way of setting permalinks. Additionally, collections have unique template variables available available (such as `path` and `output_ext`). See the ["Configuring permalinks for collections"]( ../collections#permalinks ) in Collections for more information.
+
+## Flattening pages in the _site folder on build
+
+If you want to flatten your pages (pull them out of subfolders) in the _site directory when your site builds (similar to posts), add the permalink property in each of your page's frontmatter:
+
+```
+---
+title: My page
+permalink: mypageurl.html
+---
+```
+
+Moving all pages to the root on build can make it easier to implement relative linking across your site, if that's important for you.
+
+## Extensionless permalinks with no trailing slashes {#extensionless-permalinks}
+
+Jekyll supports permalinks that contain neither a trailing slash nor a file extension, but this requires additional support from the web server to properly serve. When using extensionless permalinks, output files written to disk will still have the proper file extension (typically `.html`), so the web server must be able to map requests without file extensions to these files.
+
+Both [GitHub Pages](../github-pages/) and the Jekyll's built-in WEBrick server handle these requests properly without any additional work.
+
+### Apache
+
+The Apache web server has extensive support for content negotiation and can handle extensionless URLs by setting the [multiviews](https://httpd.apache.org/docs/current/content-negotiation.html#multiviews) option in your `httpd.conf` or `.htaccess` file:
 
 {% highlight apache %}
 Options +MultiViews
 {% endhighlight %}
 
 ### Nginx
 
-The [try_files][] directive allows you to specify a list of files to search for
-to process a request. The following configuration will instruct nginx to search
-for a file with an `.html` extension if an exact match for the requested URI is
-not found.
-
-[try_files]: http://nginx.org/en/docs/http/ngx_http_core_module.html#try_files
+The [try_files](http://nginx.org/en/docs/http/ngx_http_core_module.html#try_files) directive allows you to specify a list of files to search for to process a request. The following configuration will instruct nginx to search for a file with an `.html` extension if an exact match for the requested URI is not found.
 
 {% highlight nginx %}
 try_files $uri $uri.html $uri/ =404;
 {% endhighlight %}
+
+## Linking without regard to permalink styles
+
+You can create links in your topics to other posts, pages, or collection items in a way that is valid no matter what permalink configuration you choose. By using the `link` tag, if you change your permalinks, your links won't break. See [Linking to pages](../templates#linktags) for more details.
```

Comment:
True.. its esp. bad for SEO..  
permalinks == permanent links :stuck_out_tongue_winking_eye: